src/ecdsa/ecdh.py

```diff
@@ -97,39 +108,123 @@ def load_private_key(self, private_key):
         return self.private_key.get_verifying_key()
 
     def load_private_key_bytes(self, private_key):
+        """
+        Load private key from plain bytes string.
```

Comment:
not only, `SingingKey.from_string()` will accept any bytes-like object, not only `bytes()`